web/src/components/LandingPageTiles.tsx

```diff
@@ -0,0 +1,41 @@
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { MediaQueries } from "@/lib/PageSizes";
+import { useMediaQuery } from "@mui/material";
+import { useRouter } from "next/router";
+
+import { ReactElement } from "react";
+import { LandingPageActionTile } from "./LandingPageActionTile";
+
+export const LandingPageTiles = (): ReactElement => {
+  const router = useRouter();
+  const isMobile = useMediaQuery(MediaQueries.isMobile);
+  const { Config } = useConfig();
+
+  const autoSelectPoppyAndRouteToIndustry = () => {
```

Comment:
I think this function should be renamed; it is not fully accurate to the action that the function is performing

Reply:
updated to setFlowToStartingAndRouteToIndustry